database/javascript/2017/16041.yaml

```diff
@@ -0,0 +1,27 @@
+---
+cve: 2017-16041
+title: CVE in ikst
+description: >
+    ikst versions before 1.1.2 download resources over HTTP, which leaves it vulnerable to MITM attacks.
+cvss_v2: 4.3
+references:
+    - https://nodesecurity.io/advisories/249
+
+affected:
+    - name: ikst
+      version:
+        - "<1.1.2"
```

Comment:
Unfortunately this syntax is not supported by VictimsDB: https://github.com/victims/victims-cve-db#version-string-common 

You need to use: `"<=0.1.2"`
See the versions in https://www.npmjs.com/package/ikst